include/mega/megaclient.h

```diff
@@ -752,8 +749,12 @@ class MEGA_API MegaClient
     AsymmCipher asymkey;
 
 #ifdef USE_SODIUM
-    /// EdDSA signing key (Ed25519 privte key seed).
+
+    /// EdDSA signing key (Ed25519 private key seed).
     EdDSA signkey;
+
+    /// ECDH key (x25519 private key seed).
```

Comment:
For x25519 (Curve25519) this *is* the key, not a seed (only for Ed25519 it's a seed)